spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -425,6 +445,14 @@ def main(argv: Sequence[str]):
                                          param_centerline=param_centerline,
                                          verbose=verbose,
                                          remove_temp_files=arguments.r)
+    # Project discs labels to centerline for discfile
+    if levels_disc:
```

Comment:
Is the `levels_disc` variable necessary? Maybe we could use `if arguments.discfile`?

Reply:
I see! Thanks for the details :-)